lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -607,4 +607,10 @@ def max_thread_limit
   rescue Gem::LoadError
     puts "No gettext-setup gem found, skipping gettext initialization" if Rake.verbose == true
   end
+  namespace :module do
+  desc "Runs all tasks to build a modules POT file for internationalization"
+    task :i18n do
+      Rake::Task["gettext:metadata_pot"].invoke("#{module_dir}/metadata.json")
```

Comment:
Possibly worth while creating a ticket to ensure that when the rake task for generating the puppet strings exists, that it will be included here too.